api/views.py

```diff
@@ -291,7 +291,18 @@ def handle_post(self, request, *args, **kwargs):
         username = body['username']
         password = body['password']
 
-        user = authenticate(username=username, password=password)
+        #allowing different lower/uppercase lettered usernames:
+        caseSensitiveUsername = username
```

Comment:
@szabozoltan69 since the rest of the codebase uses `lower_case` variable and function naming, please follow that.

https://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variable-and-function-names